src/main/java/data/Account.java

```diff
@@ -15,100 +15,100 @@
-    private String accountID;
-    private String vendorID;
-    private String entity;
-    private String currency;
-    private String direction;
-    private String legacyAccountID;
-    private int nextGenAccountID;
-    private String matchingMode;
-    private String aggregationMode;
-
-    public Account(String accountID, String vendorID, String entity, 
-        String currency, String direction, String legacyAccountID, 
-        int nextGenAccountID, String matchingMode, String aggregationMode) {
-
-        this.accountID = accountID;
-        this.vendorID = vendorID;
-        this.entity = entity;
-        this.currency = currency;
-        this.direction = direction;
-        this.legacyAccountID = legacyAccountID;
-        this.nextGenAccountID = nextGenAccountID;
-        this.matchingMode = matchingMode;
-        this.aggregationMode = aggregationMode;
-    }
-
-    public String getAccountID() {
-        return accountID;
-    }
-
-    public void setAccountID(String accountID) {
-        this.accountID = accountID;
-    }
-
-    public String getVendorID() {
-        return vendorID;  
-    }
-
-    public void setVendorID(String vendorID) {
-        this.vendorID = vendorID;
-    }
-
-    public String getEntity() {
-        return entity;
-    }
-
-    public void setEntity(String entity) {
-        this.entity = entity;
-    }
-
-    public String getCurrency() {
-        return currency;
-    }
-
-    public void setCurrency(String currency) {
-        this.currency = currency;
-    }
-
-    public String getDirection() {
-        return direction;
-    }
-
-    public void setDirection(String direction) {
-        this.direction = direction;
-    }
-
-    public String getLegacyAccountID() {
-        return legacyAccountID;
-    }
-
-    public void setLegacyAccountID(String legacyAccountID) {
-        this.legacyAccountID = legacyAccountID;
-    }
-
-    public int getNextGenAccountID() {
-        return nextGenAccountID;
-    }
-
-    public void setNextGenAccountID(int nextGenAccountID) {
-        this.nextGenAccountID = nextGenAccountID;
-    }
-
-    public String getMatchingMode() {
-        return matchingMode;
-    }
-
-    public void setMatchingMode(String matchingMode) {
-        this.matchingMode = matchingMode;
-    }
-
-    public String getAggregationMode() {
-        return aggregationMode;
-    }
-
-    public void setAggregationMode(String aggregationMode) {
-        this.aggregationMode = aggregationMode;
-    }
+  private String accountID;
+  private String vendorID;
+  private String entity;
+  private String currency;
+  private String direction;
+  private String legacyAccountID;
+  private int nextGenAccountID;
+  private String matchingMode;
+  private String aggregationMode;
+
+  public Account(String accountID, String vendorID, String entity, 
+    String currency, String direction, String legacyAccountID, 
+    int nextGenAccountID, String matchingMode, String aggregationMode) {
```

Comment:
Can you indent these +2 spaces?

Reply:
It looks like line 29 & 30 already indented 2 spaces.

Reply:
Sorry, I meant two more spaces following the linewrap